app/scripts/directives/datepicker.js

```diff
@@ -36,7 +36,7 @@ angular
             : scope.localModel
             ? moment(new Date(scope.localModel)).format('MM/DD/YYYY hh:mm A')
             : null;
-        var initialDate = scope.localModel ? scope.localModel : null;
+        let initialDate = scope.localModel ? scope.localModel : null;
```

Comment:
In an ideal world, this change would have happened in a separate commit, then squashed into the prior one where the change for this happened in order to keep clean git history. Not worth the effort to change it now.

Reply:
Oh ok. Would I have needed to squash locally and then force push the changed commit?